frontend/src/ui/components/Header/Header.tsx

```diff
@@ -39,22 +67,63 @@ export const Header: React.FC = () => {
               </NavLink>
             </Route>
             <Route path={ROUTES.Console.HOME}>
-              <NavLink className="nav-link" to={ROUTES.Console.HOME}>
+              <NavLink
+                className="nav-link"
+                to={ROUTES.Console.HOME}
+                isActive={(match, location) => {
+                  // Make customize link active for any customization pages
+                  return location.pathname.startsWith(ROUTES.Console.HOME);
+                }}
+              >
                 Customize
               </NavLink>
-              <Nav.Link onClick={() => window.open(CONTACT_US_LINK, '_blank')}>
-                Support Request
-              </Nav.Link>
-              <NavLink className="nav-link" to={ROUTES.Console.NOTICE_BOARD}>
+
+              {lmsLink && (
+                <Nav.Link
+                  href={lmsLink}
+                  target="__blank"
+                  className="nav-lms-link"
+                >
+                  {deploymentPill}
+                  <span> My site (LMS) </span>
```

Comment:
We shouldn't hardcode this too - please see the other comment about `displayMessages`.

Reply:
The ``Header`` component was not using any ``displayMessages`` before. I've added ``displayMessages.ts`` in this PR and replaced all hardcoded texts with it.